src/elements/kw-share-card/kw-share-card.html

```diff
@@ -124,7 +124,7 @@
             }
         </style>
         <kw-share-shell id="shell" share="[[share]]" tombstone$="[[!share]]">
-            <div slot="share-card">
+            <div slot="share-slot">
                 <iron-image id="cover" class="cover" src="[[share.cover_url]]" sizing="cover" preload fade></iron-image>
```

Comment:
For this detailed view of the share, we don't want to only display the cover. Check how it works on kano-world at the moment